backend/src/main/java/com/crm/model/db/CustomerEntity.java

```diff
@@ -37,9 +38,12 @@
     private String name;
     private String surname;
     private String phone;
-    private String address;
 
-    @ManyToMany(cascade = CascadeType.ALL)
+    @ManyToOne(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
+    @JoinColumn(name = "address_id")
+    private AddressEntity address;
```

Comment:
Ciekawi mnie sytuacja z tymi operacjami kaskadowymi. Co w przypadku gdy klient zostanie usunięty? Nie usuwamy adresów bo jeden adres może być do kilku osób?

Reply:
Dokładnie tak to sobie obmyślłem. Wydaje mi się ok, ale jestem otwarty na sugestie.